doc/source/overview.rst

```diff
@@ -11,24 +11,70 @@ Here are the 5 levels of sharing your code. We provide tutorials for each level.
 
 .. include:: snippets/5-levels-table.rst
 
-⏩️ Are you here to **start a new Python** project?
+Where do I begin?
+-----------------
 
-    If you are **new** to programming, start from Level 1. You will learn how to reuse code across files and folders. You will also learn how to write unit tests. To get started, visit :ref:`level-1-2-3-tutorials`.
+I am here to **start a new Python** project
```

Comment:
Given

a) "Do you want to migrate your existing Python..."

b) "I want to migrate my existing Python..."

I found b) reads smoothly since reader doesn't have to convert "you" to "I" vice versa.